_posts/2019-02-14-customize.md

```diff
@@ -280,18 +280,21 @@ Note: If you have created you own front-end or are using a [third-party plug-in]
 
 ## Change the default presentation
 
-When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users.
+When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users. On a bigbluebutton update this default may become replaced .
 
 Alternatively, you can change the global default by editing `/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties` and changing the URL for `beans.presentationService.defaultUploadedPresentation`.
 
 ```properties
 # Default Uploaded presentation file
-beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/default.pdf
+beans.presentationService.defaultUploadedPresentation=${bigbluebutton.web.serverURL}/your-default.pdf
```

Comment:
the default is actually `{bigbluebutton.web.serverURL}/default.pdf`, not `{bigbluebutton.web.serverURL}/your-default.pdf`

Reply:
Ok, I was not aware that it had any function,  because I can't download the default.pdf from my server and also can't from the demo system.

Reply:
sample: http://bbb2.senfcall.de/default.pdf or https://test.bigbluebutton.org/default.pdf
This works on every BBB server, unless you set up your own / handling in your nginx config.

Reply:
ok, I see. 

we can close my PR. I will investigate my configuration.